### PR TITLE
Update installation guide

### PR DESCRIPTION
**Pull request by LeonardoMariano**
The dojot installation guide (for both docker-compose and k8s) has been updated to meet the needs of version v0.5.0

### CODE REVIEW

**Review by mprevide — CHANGES_REQUESTED** [review]
You also need to translate to pt-br.

File: source/locale/pt_BR/LC_MESSAGES/installation-guide.po [hunk 1]
```diff
@@ -104,56 +104,87 @@ msgstr ""
...
+msgid ""
+"**TLS**: 8883 *(MQTTS, If you are going to use MQTT with TLS, in secure "
+"mode.)*."
+msgstr ""
+
+#: ../../source/installation-guide.rst:52
+msgid ""
+"**UDP**: 5683 and 5693 *(LW2M, If you are going to use LW2M)*; 5684 and "
+"5694 *(LW2M, If you are going to use LW2M with DTLS)*."
+msgstr ""
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/installation-guide.po:144]: MIssing translation

File: source/locale/pt_BR/LC_MESSAGES/installation-guide.po [hunk 2]
```diff
@@ -104,56 +104,87 @@ msgstr ""
...
 "use MQTT)*; 5896 *(LW2M, If you are going to use LW2M file server via "
 "HTTP instead of coap, UDP)*."
 msgstr ""
 
-#: ../../source/installation-guide.rst:45
+#: ../../source/installation-guide.rst:51
+msgid ""
+"**TLS**: 8883 *(MQTTS, If you are going to use MQTT with TLS, in secure "
+"mode.)*."
+msgstr ""
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/installation-guide.po:138]: MIssing translation

File: source/locale/pt_BR/LC_MESSAGES/installation-guide.po [hunk 3]
```diff
@@ -104,56 +104,87 @@ msgstr ""
...
+#: ../../source/installation-guide.rst:54
+msgid "The following ports should be opened on load balancer for kubernetes deployment:"
+msgstr "As seguintes portas devem estar abertas no balanceador de carga para a instalação com kubernetes:"
+
+#: ../../source/installation-guide.rst:55
+msgid ""
+"**TCP**: 80 *(web interface access)*; 1883 *(MQTT, If you are going to "
+"use MQTT)*; 5896 *(LW2M, If you are going to use LW2M file server via "
+"HTTP instead of coap, UDP)*."
+msgstr ""
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/installation-guide.po:155]: MIssing translation

File: source/locale/pt_BR/LC_MESSAGES/installation-guide.po [hunk 4]
```diff
@@ -104,56 +104,87 @@ msgstr ""
...
+
+#: ../../source/installation-guide.rst:56
 msgid ""
 "**TLS**: 8883 *(MQTTS, If you are going to use MQTT with TLS, in secure "
 "mode.)*."
 msgstr ""
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/installation-guide.po:161]: MIssing translation

File: source/locale/pt_BR/LC_MESSAGES/installation-guide.po [hunk 5]
```diff
@@ -104,56 +104,87 @@ msgstr ""
...
 
-#: ../../source/installation-guide.rst:46
+#: ../../source/installation-guide.rst:57
 msgid ""
 "**UDP**: 5683 and 5693 *(LW2M, If you are going to use LW2M)*; 5684 and "
 "5694 *(LW2M, If you are going to use LW2M with DTLS)*."
 msgstr ""
```
Comment (mprevide) [inline source/locale/pt_BR/LC_MESSAGES/installation-guide.po:167]: MIssing translation

**Review by mprevide — APPROVED** [review]
Great :1st_place_medal:

**Comment by mprevide** [comment]
Do you see me suggestions https://github.com/dojot/dojot/issues/1473?

File: source/installation-guide.rst [hunk 6]
```diff
@@ -153,180 +168,47 @@ instance: ::
...
 
-- The Kubernetes supported version is between **v1.11** and **v1.15** .
-- Access to Docker Hub repositories
-- (optional) a storage class that will be used for persistent storage
+When we deploy dojot with the command 'docker-compose up -d' the volumes are enabled and created
+by default.
 
-dojot Deployment
-^^^^^^^^^^^^^^^^
+The volumes of microservices that Dojot uses are incompatible between dojot versions. This means
```
Comment (andersonluisribeiro) [inline source/installation-guide.rst:177]: I think they "can be" incompatible

**Review by andersonluisribeiro — CHANGES_REQUESTED** [review]

**Review by andersonluisribeiro — APPROVED** [review]
:+1:

**Review by mprevide — APPROVED** [review]

**Pull request merged**